dist/leanplum.d.ts

```diff
@@ -0,0 +1,100 @@
+        static setSystemVersion(systemVersion: string): void;
+        static setVariables(variables: Object): void;
+        static setRequestBatching(batchEnabled: boolean, cooldownSeconds: number): void;
+        static getVariables(): any;
+        static getVariable(...args: string[]): any;
+        static getVariants(): any[];
+        static addStartResponseHandler(handler: StatusHandler): void;
+        static addVariablesChangedHandler(handler: SimpleHandler): void;
+        static removeStartResponseHandler(handler: StatusHandler): void;
+        static removeVariablesChangedHandler(handler: SimpleHandler): void;
+        static forceContentUpdate(callback: StatusHandler): void;
+        static start(userId: string, callback: StatusHandler): void;
+        static start(userAttributes?: UserAttributes, callback?: StatusHandler): void;
+        static startFromCache(userId: string, callback: StatusHandler): void;
+        static startFromCache(userAttributes?: UserAttributes, callback?: StatusHandler): void;
+        static stop(): void;
+        static pauseSession(): void;
+        static resumeSession(): void;
+        static pauseState(): void;
+        static resumeState(): void;
+        static setUserId(userId: string): void;
+        static setUserAttributes(userId: string, userAttributes?: UserAttributes): void;
+        static track(event: string, value: number, params: Object): void;
+        static track(event: string, params: Object): void;
+        static advanceTo(state: string, params?: Object): void;
+        /**
+            * Determines if web push is supported in the browser.
+            * @return {Boolean} True if supported, else false.
+            */
+        static isWebPushSupported(): boolean;
+        /**
+            * Determines if web push is subscribed.
+            * @return {Promise} Resolves if true, rejects if false.
+            */
+        static isWebPushSubscribed(): Promise<boolean>;
+        /**
+            * Register the browser for web push.
+            * @param  {string}   serviceWorkerUrl The url on your server that hosts the
+            *                                     /sw.min.js service worker js file.
+            * @return {Promise}                   Resolves if registration successful,
+            *                                     otherwise fails.
+            */
+        static registerForWebPush(serviceWorkerUrl: string): Promise<boolean>;
+        /**
+            * Unregisters the browser form web push.
+            * @return {Promise}            Resolves on success, otherwise rejects.
+            */
+        static unregisterFromWebPush(): Promise<string>;
+        /**
+            * Clears cached values for messages, variables and test assignments.
+            * Use sparingly as if the app is updated, you'll have to deal with potentially
+            * inconsistent state or user experience.
+            */
+        static clearUserContent(): void;
```

Comment:
nit: I'm not sure if this is auto-generated but it seems that there's too much indentation (and misaligned comments).

Reply:
It's auto-generated from [this code](https://github.com/Leanplum/Leanplum-JavaScript-SDK/blob/8d1c4909aedf35e549f34d293545d248fd773ace/src/Leanplum.ts#L389-L392), and it seems that the generation doesn't align the comments.